include/swift/AST/Module.h

```diff
@@ -200,6 +200,10 @@ class ModuleDecl : public DeclContext, public TypeDecl {
 
   SmallVector<FileUnit *, 2> Files;
 
+  /// The parseable interface this module was generated from if any.
+  /// Used for debug info.
+  std::string ParseableInterface;
```

Comment:
This definitely doesn't belong here. Maybe in SerializedASTFile, possibly with an API on FileUnit?

Reply:
Done.